### PR TITLE
Use absolute target names in glfw

### PR DESCRIPTION
**Pull request by tapia**
Conan feature related: https://github.com/conan-io/conan/pull/10099

### CODE REVIEW

File: recipes/glfw/all/conanfile.py [hunk 1]
```diff
@@ -129,7 +129,7 @@ def _module_file_rel_path(self):
 
     def package_info(self):
         self.cpp_info.set_property("cmake_file_name", "glfw3")
-        self.cpp_info.set_property("cmake_target_name", "glfw")
+        self.cpp_info.set_property("cmake_target_name", "glfw::glfw")
```
Comment (SpaceIm) [inline recipes/glfw/all/conanfile.py:132]:
```suggestion
        self.cpp_info.set_property("cmake_target_name", "glfw")
```

File: recipes/glfw/all/conanfile.py [hunk 2]
```diff
@@ -129,7 +129,7 @@ def _module_file_rel_path(self):
...
         self.cpp_info.set_property("pkg_config_name", "glfw3")
         self.cpp_info.builddirs.append(self._module_subfolder)
         self.cpp_info.set_property("cmake_build_modules", [self._module_file_rel_path])
```
Comment (SpaceIm) [inline recipes/glfw/all/conanfile.py:135]:
```suggestion
```

**Review by SpaceIm — CHANGES_REQUESTED** [review]
transition will be funny ^^

(glfw imported target is not namespaced, so we want `glfw` for `cmake_target_name` with the new model, and we don't want to add the custom module in `CMakeDeps` because it's a workaround to provide non-namespaced imported target to `cmake_find_package_*`)

File: recipes/glfw/all/conanfile.py (hunk 2 above)
Comment (tapia) [reply recipes/glfw/all/conanfile.py]: Fixed (?)

**Review by SpaceIm — DISMISSED** [review]

**Comment by SSE4** [comment]
@tapia there are conflicts, please resolve

**Comment by SpaceIm** [comment]
Just bump required_conan_version and remove TODO above

**Review by czoido — APPROVED** [review]

**Pull request closed without merging**